ci-operator/step-registry/optional-operators/ci/subscribe/optional-operators-ci-subscribe-run-bundle.yaml

```diff
@@ -0,0 +1,32 @@
+ref:
+  as: optional-operators-ci-run-bundle
+  from: cli
+  commands: optional-operators-ci-run-bundle-commands.sh
```

Comment:
Does this still need to be a shell script? Can't this be `operator-sdk run bundle ... `?

Reply:
It has to be a shell script I think as it will be injected into the image.